sites/healthcarepackaging/index.js

```diff
@@ -7,6 +7,7 @@ const coreConfig = require('./config/core');
 const document = require('./server/components/document');
 const components = require('./server/components');
 const fragments = require('./server/fragments');
+const redirect = require('./redirect-handler');
```

Comment:
Import this as `const redirectHandler = require('./redirect-handler');`

This change applies to all files.